cli/binary/list.go

```diff
@@ -108,10 +65,14 @@ func parseBinaries(fileList []fs.DirEntry, programName string,
 		if strings.HasPrefix(f.Name(), versionPrefix) {
 			versionStr := strings.TrimPrefix(strings.TrimPrefix(f.Name(), versionPrefix), "v")
 			if versionStr == "master" {
-				binaryVersions = append(binaryVersions, version.Version{
-					Major: math.MaxUint, // Small hack to make master the newest version.
-					Str:   "master",
-				})
+				ver := version.Version{}
+				ver.Major = math.MaxUint // Small hack to make master the newest version.
+				if binActive == f.Name() {
```

Comment:
We can make this code common for all versions. Now it is duplicate.

Reply:
changed